docs/sections/user_guide/yaml/tags.rst

```diff
@@ -0,0 +1,76 @@
+.. _defining_YAML_tags:
+
+Defining YAML ``tags``
+==========================
+
+Tags are used to denote the type of a YAML node when converting to a Python object. Standard YAML tags are defined at http://yaml.org/type/index.html.
+
+Tags may be implicit:
+
+.. code-block:: yaml
+
+   boolean: true
+   integer: 3
+   float: 3.14
+
+Or explicit:
+
+.. code-block:: yaml
+
+   boolean: !!bool "true"
+   integer: !!int "3"
+   float: !!float "3.14"
+
+Additionally, UW extends the following tags and constructors to support other formats:
+
+UW YAML Tags
+------------
+
+``!float``
+^^^^^^^^^^
+
+Converts the tagged node to a Python ``float`` type.
+
+.. code-block:: yaml
+
+   f2: !float '3.14'
```

Comment:
```suggestion
   f2: !float "{{ 3.141 + 2.718 }}"
```
Suggested because the current value could be handled by the built-in `!!float`, while the suggest value could not.